daemons/compute/client-cpp/client.cc

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2022 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <iostream>
+#include <memory>
+#include <string>
+
+#include <grpcpp/grpcpp.h>
+
+#include "datamovement.grpc.pb.h"
+
+using grpc::Channel;
+using grpc::ClientContext;
+using grpc::Status;
+using datamovement::DataMover;
+using datamovement::DataMovementCreateRequest;
+using datamovement::DataMovementCreateResponse;
+using datamovement::DataMovementCreateResponse_Status;
+using datamovement::DataMovementStatusRequest;
+using datamovement::DataMovementStatusResponse;
+using datamovement::DataMovementStatusResponse_Status;
+using datamovement::DataMovementDeleteRequest;
+using datamovement::DataMovementDeleteResponse;
+using datamovement::DataMovementDeleteResponse_Status;
```

Comment:
Delete and Status could be switched around to alphabetize

Reply:
I prefer "order of use"